internal/term/term.go

```diff
@@ -0,0 +1,16 @@
+package term
+
+import (
+	"os"
+	"strings"
+)
+
+// SupportsProgressBar tries to determine whether the current terminal supports
+// progress bars by looking into environment variables.
+func SupportsProgressBar() bool {
+	termProg := os.Getenv("TERM_PROGRAM")
+	_, wtSessionOk := os.LookupEnv("WT_SESSION")
```

Comment:
I'd consider `hasWtSession` or `isWindowsTerminal` for name:

```suggestion
	_, isWindowsTerminal := os.LookupEnv("WT_SESSION")
```